### PR TITLE
added calender

### PR DESCRIPTION
**Pull request by bethea28**

### CODE REVIEW

**Review by kaladin9017 — CHANGES_REQUESTED** [review]
Just add the calendar file, the clock file, and css file. It should be inside the style folder.

File: public/bundle.js [hunk 1]
```diff
@@ -68,7 +68,7 @@
```
Comment (nmadd) [inline public/bundle.js:68]: add your bundle file to .gitignore. you don't want them showing up in pull requests

File: src/components/widgets/calender/calender.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    date: '' ,
+    day: '',
+    month:'',
+    year: '',
+    daysArray:''
+   }
+ },
+
+ componentDidMount(){
+
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:19]: Clean this up! Formatting is a mess

File: src/components/widgets/calender/calender.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  var date = new Date()
+  var daysArray = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"]
+  var dayNumber = date.getDate()
+  var day = daysArray[date.getDay()]
+  var monthArray = ['January', 'February', 'March','April', 'May', 'June', 'July', 'August', 'September', 'October','November','December']
+  var month = monthArray[date.getMonth()]
+  var year = date.getFullYear()
+
+  this.setState({ date:date, month: month, day: day, year: year, dayNumber: dayNumber})
+
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:29]: remove all console.logs before opening a pull request. console.logs shouldn't get merged into your main repo

File: src/components/widgets/calender/calender.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  console.log(day)
+  console.log(this.state)
+
+ },
+
+ render(){
+  console.log(this.state)
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:36]: remove console.log

File: src/components/widgets/calender/calender.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+          <Clock/>
+         </center>
+         
+        </ul>
+     </div>
+
+    </div>
+   )
+  }else{
+   return(
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:81]: fix formatting

File: src/components/widgets/calender/calender.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+        </ul>
+
+        <ul style = {{height: 100}}className="days"> 
+
+         <center> 
+          <li >{this.state.dayNumber}</li>
+         </center>
+
+         <center>
+          <Clock/>
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:72]: fix formatting

File: src/components/widgets/calender/clock.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+const Clock = React.createClass({
+ getInitialState(){
+  return(
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:5]: fix formatting

File: src/components/widgets/calender/clock.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+    {
+     hour: '',
+     minutes: '',
+     seconds: ''
+    }
+  )
+ },
+
+ componentDidMount(){
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:14]: fix formatting

File: src/components/widgets/calender/clock.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+  var today = new Date();
+  var hour = today.getHours();
+  var minutes = today.getMinutes();
+  var seconds = today.getSeconds();
+  var startTime = () =>{
+   for(var i = 0; i < 0; i++){
+
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:21]: fix formatting

File: src/components/widgets/calender/clock.js [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+  var startTime = () =>{
+   for(var i = 0; i < 0; i++){
+    
+   
+   }
+  }
+   
+   this.setState({
+    hour: hour, minutes: minutes, seconds: seconds
+   })
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:28]: remove commented out code before opening a pull request. commented out code should not get merged into master

File: src/components/widgets/calender/clock.js [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+// </script>
+// </head>
+
+// <body onload="startTime()">
+
+// <div id="txt"></div>
+
+// </body>
+ render(){
+  console.log(this.state)
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:56]: remove console.log

File: src/components/widgets/calender/clock.js [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+  if(this.state){
+
```
Comment (nmadd) [inline src/components/widgets/calender/clock.js:58]: fix formatting

File: src/components/widgets/calender/calender.js [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+  return(
+   <div style = {{display: 'flex', justifyContent: 'center'}}>
+     <div style = {{width: 450}}>
+
+        <div style = {{width: 400, borderRadius: 5}}className="month"> 
+          <ul>
+            <li className="prev"></li>
+            <li className="next"></li>
+
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:47]: fix formatting

File: src/components/widgets/calender/calender.js [hunk 14]
```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import css from './calender.css'
+import Clock from './clock'
+
+const Calender = React.createClass({
+
+ getIniitalState(){
+
```
Comment (nmadd) [inline src/components/widgets/calender/calender.js:8]: fix formatting

**Pull request closed without merging**